Lib/test/test_dis.py

```diff
@@ -1198,6 +1198,17 @@ def test_show_caches(self):
                     self.assertEqual(caches.count(""), empty_caches)
                     self.assertEqual(len(caches), total_caches)
 
+    @cpython_only
+    def test_show_currinstr_with_cache(self):
+        def f():
+            print(a)
+        # The code above should generate a LOAD_GLOBAL which has CACHE instr after
+        # Make sure that with lasti pointing to CACHE, it still shows the current
+        # line correctly
```

Comment:
This test is making assumptions: (1) the code would have a LOAD_GLOBAL (2) LOAD_GLOBALS has a cache of size at least something.

Both these assertions could change, and then the test will fail. It will be good to explicitly assert these assumptions, so that it will be clear why the test is failing.